Java/Sorting Algorithm/TreeSort.java

```diff
@@ -0,0 +1,75 @@
+public class TreeSort {
```

Comment:
Just a minor:
Add the comment on the top of the file:
`Tree Sort Implementation` or something like this.

Reply:
Done